src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2840,6 +2840,213 @@
+  - name: Create a deployment stack with parameter file and delete resources.
+    text: az stack mg create --name StackName --management-group-id myMg --delete-resources --template-file simpleTemplate.json --parameters simpleTemplateParams.json --location westus2 --description description --deny-settings-mode None
+  - name: Create a deployment stack with template spec and delete resource groups
+    text: az stack mg create --name StackName --management-group-id myMg --delete-resource-groups --template-spec TemplateSpecResourceIDWithVersion --location westus2 --description description --deny-settings-mode None
+  - name: Create a deployment stack using bicep file and delete all resources.
+    text: az stack mg create --name StackName --management-group-id myMg --delete-all --template-file simple.bicep --location westus2 --description description --deny-settings-mode None
+  - name: Create a deployment stack using parameters from key/value pairs
+    text: az stack mg create --name StackName --management-group-id myMg --template-file simpleTemplate.json --location westus --description description --parameters simpleTemplateParams.json value1=foo value2=bar --deny-settings-mode None
+  - name: Create a deployment stack from a local template, using a parameter file, a remote parameter file, and selectively overriding key/value pairs.
+    text: az stack mg create --name rollout01 --management-group-id myMg --template-file azuredeploy.json --parameters @params.json --parameters https://mysite/params.json --parameters MyValue=This MyArray=@array.json --location westus --deny-settings-mode None
+  - name: Create a deployment stack from a local template, using deny settings.
+    text: az stack mg create --name rollout01 --management-group-id myMg --template-file azuredeploy.json --deny-settings-mode denyDelete --deny-settings-excluded-actions Microsoft.Compute/virtualMachines/write --deny-settings-excluded-principals "test1 test2" --location westus --deny-settings-mode None
+  - name: Create a deployment stack from a local template, apply deny settings to child scope.
+    text: az stack mg create --name rollout01 --management-group-id myMg --template-file azuredeploy.json --deny-settings-mode denyDelete --deny-settings-excluded-actions Microsoft.Compute/virtualMachines/write --deny-settings-apply-to-child-scopes --location westus --deny-settings-mode None
+"""
+
+helps['stack mg list'] = """
+type: command
+short-summary: List all deployment stacks in management group
+examples:
+  - name: List all stacks
+    text: az stack mg list --management-group-id myMg
+"""
+
+helps['stack mg show'] = """
+type: command
+short-summary: Get specified deployment stack from management group scope
+examples:
+  - name: Get stack by name.
+    text: az stack mg show --name StackName --management-group-id myMg
+  - name: Get stack by stack resource id.
+    text: az stack mg show --id StackResourceID --management-group-id myMg
```

Comment:
It is better to specific the format of `--id`, like `--id /subscriptions/0b1f6471-1bf0-4dda-aec3-111111111111/resourceGroups/MyResourceGroup/providers/Microsoft.Network/virtualNetworks/MyVnet/subnets/MySubnet`